cli/command/trust/inspect.go

```diff
@@ -76,9 +76,8 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	if err != nil {
 		logrus.Debug(trust.NotaryError(ref.Name(), err))
 		// print an empty table if we don't have signed targets, but have an initialized notary repo
-		if !strings.Contains(err.Error(), "No valid trust data for") {
-			return fmt.Errorf("No signatures or cannot access %s", remote)
-		}
+		return fmt.Errorf("No signatures or cannot access %s", remote)
```

Comment:
We should be checking for `client. ErrRepositoryNotExist` here and only erroring out on that case.  Something like this should do the trick:

```
if _, ok := err.(client.ErrRepositoryNotExist); ok {
...
```

`client.ErrNoSuchTarget` should be OK to continue on because in that scenario we have keys/signers that we would want to display.